### PR TITLE
Law Enforcement

### PR DESCRIPTION
**Pull request by miriamseely**
Current draft of the metadata for the [Utah Law Enforcement layer](https://gis.utah.gov/products/sgid/society/law-enforcement-facilities/).

### CODE REVIEW

File: metadata/society/law_enforcement [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Title
+
+Utah Law Enforcement
+
+<!--- thought: should the title say "Utah Law Enforcement Facilities"? That's what it says in the link, but on the data page it is just "Utah Law Enforcement" which seems vague to me. -->
```
Comment (mhchou) [inline metadata/society/law_enforcement:5]: I agree.

File: metadata/society/law_enforcement [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+01008931-c755-4df7-aea0-b71aded68f5a
+
+## Brief Summary
+
+Point locations of facilities where sworn officers of a law enforcement agency are regularly stationed.
+
+## Summary
+
+Utah Law Enforcement contains locations where law enforcement officers are regularly based or stationed. Law enforcement agencies may include municipal police, county sheriffs, school police, and federal law enforcement agencies. Additionally, this dataset includes departments within non-law enforcement federal agencies charged with law enforcement (e.g., US Postal Inspectors), and cross jurisdictional authorities (e.g., Port Authority Police).
```
Comment (mhchou) [inline metadata/society/law_enforcement:17]: + Facilities

File: metadata/society/law_enforcement [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+This dataset was created to provide a general purpose database of law enforcement facility locations. Law enforcement facility data can been used to asses whether the total police capability of a given area is adequate and to help local governments identify areas of greatest concern. The data are also a helpful resource for situational awareness planning, emergency response, and route analysis.
+
+### What does the dataset represent?
+
+This dataset is comprised of point data indicating the locations of law enforcement facilities throughout the state of Utah. Each point includes attributes such as the facility name, street address, and date it was added to the dataset (indicated by the CONTDATE field). Points have been placed on or near the main structure of the facility.
+
+### How was the dataset created?
+
+<!--- Gathering more information on this. -->
```
Comment (mhchou) [inline metadata/society/law_enforcement:37]: @gregbunce Do you have more info on this?

Comment (gregbunce) [reply metadata/society/law_enforcement]: The law enforcement boundaries are created programmatically using a [python script](https://github.com/gregbunce/NextGen911UtahGISDataLoader/blob/798e52335e69af2f1a761aa245fc6ecfe619f9b6/NextGen911DataLoader/commands/LoadLawEnforcement.cs). The boundaries are composed of SGID County boundaries and SGID Municipal boundaries. The sheriff offices are built from County boundaries and the police depts are built from the Muni boundaries.

Comment (miriamseely) [reply metadata/society/law_enforcement]: Thanks for the info! Just want to make sure--does this all refer to the law enforcement boundaries dataset as well as this one, or just the law enforcement polygon boundaries? (this pull request is for the law enforcement facilities points) Is that to say that the sheriff's office points on this layer are derived from the county boundary polygons, and the police dept points from the Muni boundaries?

Comment (gregbunce) [reply metadata/society/law_enforcement]: ahh, good question. yes, you are correct, the data process I outlined above is solely for the polygon boundaries (not this layer). thanks for keeping me in line on that.

**Comment by gregbunce** [comment]
i'll need to ask around how this dataset has been created. my guess is that an individual from ugrc (years ago) worked with the psaps to add facilities relevant to the 911 community.

File: metadata/society/law_enforcement (hunk 3 above)
Comment (miriamseely) [reply metadata/society/law_enforcement]: Good to know! I'll jot this down and add it in once I get to the law enforcement boundaries dataset.

**Comment by miriamseely** [comment]
> i'll need to ask around how this dataset has been created. my guess is that an individual from ugrc (years ago) worked with the psaps to add facilities relevant to the 911 community.

Sounds good. Let me know what you find out! And, if you have specific people in mind you are thinking of asking about it, I can also reach out to them if you would prefer. Whatever works for you!

File: metadata/society/law_enforcement.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Utah Law Enforcement Facilities
+
+## ID
+
+01008931-c755-4df7-aea0-b71aded68f5a
+
+## Brief Summary
+
+Point locations of facilities where sworn officers of a law enforcement agency are regularly stationed.
```
Comment (jacobdadams) [inline metadata/society/law_enforcement.md:11]:
```suggestion
Point locations of facilities where sworn law enforcement officers are regularly stationed.
```

File: metadata/society/law_enforcement.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+The US Bureau of Justice Statistics describes law enforcement agencies as "publicly funded and employ at least one full-time or part-time sworn officer with general arrest powers". This dataset includes point locations for federal, state, local, and special jurisdiction law enforcement agency facilities.
+
+### What is the purpose of the dataset?
+
+This dataset was created to provide a general purpose database of law enforcement facility locations. Law enforcement facility data can been used to asses whether the total police capability of a given area is adequate and to help local governments identify areas of greatest concern. The data are also a helpful resource for situational awareness planning, emergency response, and route analysis.
+
+### What does the dataset represent?
+
+The points in this dataset indicate the locations of law enforcement facilities throughout the state of Utah. Each point includes attributes such as the facility name, street address, and date it was added to the dataset (indicated by the CONTDATE field). Points have been placed on or near the main structure of the facility.
```
Comment (jacobdadams) [inline metadata/society/law_enforcement.md:29]:
```suggestion
The points in this dataset indicate the locations of law enforcement facilities (police stations, sheriff's offices, etc) throughout the state of Utah. Each point includes attributes such as the facility name, street address, and date it was added to the dataset (indicated by the CONTDATE field). Points have been placed on or near the main structure of the facility.
```
Trying to sus out what constitutes a "facility" a little more.

File: metadata/society/law_enforcement.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+### How was the dataset created?
+
+UGRC derived this dataset from the Homeland Infrastructure Foundation-Level Data (HIFLD) database as maintained by the [US Bureau of Justice Statistics](https://bjs.ojp.gov/). The primary source for this database was the 2008 [Census of State and Local Law Enforcement Agencies](https://bjs.ojp.gov/library/publications/census-state-and-local-law-enforcement-agencies-2008). This dataset is currently maintained by UGRC in collaboration with local and state government authorities.
```
Comment (jacobdadams) [inline metadata/society/law_enforcement.md:33]:
```suggestion
UGRC derived this dataset from the Homeland Infrastructure Foundation-Level Data (HIFLD) database maintained by the [US Bureau of Justice Statistics](https://bjs.ojp.gov/). The primary source for this database was the 2008 [Census of State and Local Law Enforcement Agencies](https://bjs.ojp.gov/library/publications/census-state-and-local-law-enforcement-agencies-2008). Our dataset is currently maintained by UGRC in collaboration with local and state government authorities.
```
Replacing "this" with "our" to be a little more clear about which dataset (HIFLD, 2008 census, our layer) we're talking about.

File: metadata/society/law_enforcement.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+### How reliable and accurate is the dataset?
+
+This dataset previously included home addresses for constables who work out of their private residence, temporary offices, and FBI entities. The most recent version of this dataset excludes these facilities, however, some discrepancies may still exist. Please reach out to [our team](https://gis.utah.gov/contact/) with questions concerning this dataset.
```
Comment (jacobdadams) [inline metadata/society/law_enforcement.md:37]:
```suggestion
This dataset previously included home addresses for constables who work out of their private residence, temporary offices, and FBI entities. The most recent version of this dataset excludes these facilities; however, some discrepancies may still exist. Please reach out to [our team](https://gis.utah.gov/contact/) with questions concerning this dataset.
```

**Review by jacobdadams — APPROVED** [review]
Looks good, just a few minor grammatical changes.

**Pull request merged**